deb/openmediavault/var/www/openmediavault/js/omv/grid/Panel.js

```diff
@@ -95,7 +95,10 @@ Ext.define("OMV.grid.Panel", {
 			var fnActivateTask = function() {
 				if (Ext.isEmpty(me.reloadTask)) {
 					me.reloadTask = Ext.util.TaskManager.newTask({
-						run: me.doReload,
+						run: function() {
+							if(!me.getStore().isLoading())
```

Comment:
in my opinion this check should be done in doReload().

Reply:
It would be prettier but i wanted only to change autoreload feature behavior to avoid side effect with other calls to doReload (it is my first read of the source :-)). 

So I should do the checking in doReload() but in the derived class instead (omv/workspace/grid/Panel.js) ?

Reply:
You can strip this commit from your PR because https://github.com/openmediavault/openmediavault/commit/7ae15b03ac177ee373055120c3314b10a747a7e2 will fix that issue.

Reply:
Could you cherry-pick your correction https://github.com/openmediavault/openmediavault/commit/7ae15b03ac177ee373055120c3314b10a747a7e2 to 3.x in your next release ?
Thanks.

Reply:
This is already done: https://github.com/openmediavault/openmediavault/commit/75a8a83755429b87c3e422dc58fa8282e229d420

Reply:
Done, see https://github.com/openmediavault/openmediavault/commit/74273141a39e3be90498fd3fe3a66a40641da98e